larray_editor/editor.py

```diff
@@ -272,13 +263,13 @@ def setup_menu_bar(self):
         file_menu.addAction(create_action(self, _('Save Data &As'), triggered=self.save_data_as,
                                           statustip=_('Save all arrays as a session in a file')))
         recent_files_menu = file_menu.addMenu("Open &Recent Data")
-        for action in self.recent_file_actions:
+        for action in self.recent_data_files.actions:
             action.setVisible(False)
             action.triggered.connect(self.open_recent_file)
             recent_files_menu.addAction(action)
```

Comment:
move these two lines to `RecentFileList.__init__`?

Reply:
Or create the list of actions here and call the `actions` setter after the loop.